src/figuresClasses.ts

```diff
@@ -1,19 +1,73 @@
-export interface Figure {
+type Colors = 'red' | 'blue' | 'green';
+type Shape = 'circle' | 'triangle' | 'rectangle';
 
+export interface Figure {
+  shape: Shape;
+  color: Colors;
+  getArea(): number;
 }
 
-export class Triangle {
+export class Triangle implements Figure {
+  shape: Shape = 'triangle';
 
-}
+  constructor(
+    public color: Colors,
+    public a: number,
+    public b: number,
+    public c: number,
+  ) {
+    const maxSide: number = Math.max(a, b, c);
 
-export class Circle {
+    if (maxSide >= a + b + c - maxSide) {
+      throw new Error('Sides should be positive');
+    }
+  }
 
+  getArea(): number {
+    const p:number = (this.a + this.b + this.c) / 2;
+    const
+      area:number = Math.sqrt(p * (p - this.a) * (p - this.b) * (p - this.c));
+
+    return Math.floor(100 * area) / 100;
```

Comment:
you round down to hundredths on each getArea, maybe better create a separate method for it?